lib/tasks/demo_data_for_development.rake

```diff
@@ -0,0 +1,70 @@
+namespace :data do
+  desc 'Create demo data for our local development'
+  task test: :environment do
+    include FactoryGirl::Syntax::Methods
+
+    def generate_events program, conference
```

Comment:
Since we need the events for all conferences, so I thought it would be better to do it with a method rather than writing the same code to generate events for all the conferences.